examples/AdminClient/Program.cs

```diff
@@ -84,7 +85,202 @@ static async Task CreateTopicAsync(string bootstrapServers, string topicName)
                 }
                 catch (CreateTopicsException e)
                 {
-                    Console.WriteLine($"An error occured creating topic {e.Results[0].Topic}: {e.Results[0].Error.Reason}");
+                    Console.WriteLine($"An error occurred creating topic {e.Results[0].Topic}: {e.Results[0].Error.Reason}");
```

Comment:
hey, i thought i was the native english speaker 😉

Reply:
It's the automatic spell check I've installed to avoid my errors :smile: